intelmq/bots/collectors/shadowserver/collector_reports_api.py

```diff
@@ -36,6 +39,8 @@ class ShadowServerAPICollectorBot(CollectorBot, HttpMixin, CacheMixin):
     api_key = None
     secret = None
     types = None
+    reports = None
+    report_list = []
```

Comment:
```suggestion
    _report_list = []
```
Looks like it is an internal variable?

Reply:
Fixed:
```
-      report_list = []
+      _report_list = []
```